README.rst

```diff
@@ -10,6 +10,13 @@ the Symfony Core Team. For contributed recipes, see the `contrib repository`_.
 Creating Recipes
 ----------------
 
+1. Clone a repository
```

Comment:
Hm, technically it's not required to _clone_ the repo, you can use GitHub GUI for this

Reply:
@bocharsky-bw This is error , I wanted to write "fork" instead "clone". I have fixed it now.